src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -40,7 +40,7 @@ const propTypes = {
                 attributes: PropTypes.shape({
                     unit: PropTypes.string,
                 }),
-                rates: PropTypes.arrayOf(
+                rates: PropTypes.objectOf(
```

Comment:
An issue I noticed while testing updating a custom unit is that the Onyx data returned [here](https://github.com/Expensify/Web-Expensify/blob/main/lib/PolicyAPI.php#L2999) doesn't have the correct shape for rates data, so that results in a prop types error here:

<img width="1061" alt="Screen Shot 2022-08-30 at 3 20 13 PM" src="https://user-images.githubusercontent.com/7277067/187463809-1e10b07d-6c39-4299-8eda-b0abbbde47e8.png">

Not sure if it makes sense for you to include it in https://github.com/Expensify/Web-Expensify/pull/34720 or if we should make a follow up issue to handle separately, just wanted to raise it somewhere for us collectively 😄

Reply:
Totally think it makes sense to fix this in the Web-E PR, especially after Chirag's comments [here](https://github.com/Expensify/Web-Expensify/pull/34720#discussion_r958958133)

Reply:
Thanks so much for catching this!

Reply:
Just updated [Expensify/Web-Expensify#34720](https://github.com/Expensify/Web-Expensify/pull/34720) to fix this if you wanna take a look

Reply:
Cool, thanks for doing that!